Primo.js

```diff
@@ -530,6 +613,63 @@ var testCases = [
 				"seeAlso": []
 			}
 		]
-	}
+	},
+	{
+		"type": "web",
+		"url": "http://digitale.beic.it/primo_library/libweb/action/display.do?dscnt=1&fromLogin=true&doc=39bei_digitool4783627&dstmp=1420050573310&vid=beic&fromLogin=true&fromLogin=true",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "De scientia stellarum",
+				"creators": [
+					{
+						"firstName": "Muhammad",
+						"lastName": "al-Battani",
+						"creatorType": "author"
+					}
+				],
+				"date": "1645",
+				"language": "lat",
+				"libraryCatalog": "Primo",
+				"numPages": "16+228+4",
+				"publisher": "Vittorio Benacci",
+				"attachments": [],
+				"tags": [
+					"Stelle fisse - Novae"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://digitale.beic.it/primo_library/libweb/action/display.do?doc=39bei_digitool2018516",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Grida per i Milanesi che avevano seguito Ludovico il Moro",
+				"creators": [
+					{
+						"lastName": "Milano",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
+				"date": "1500",
+				"language": "ita",
+				"libraryCatalog": "Primo",
+				"place": "Milano",
+				"publisher": "Ambrogio da Caponago",
+				"attachments": [],
+				"tags": [
+					"Italia - Storia medioevale",
+					"Leggi"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
```

Comment:
Aurimas Vinckevicius, 04/05/2015 17:19:
> More specifically, this is your problem, but that tells me that you
> didn't run through tests before submitting the commit. Please always run
> all tests.

Sorry, I had only run them before all the rebases. They now all succeed, 
except libis.be and princeton.edu which don't respond.